frontend/src/modules/EditZing/Components/EmailModal.tsx

```diff
@@ -101,6 +237,28 @@ export const EmailModal = ({
     }
   }
 
+  const SelectTemplates = () => {
+    return (
+      <>
+        {' '}
```

Comment:
lmao the space character `{' '}`, is your editor configured to delete trailing whitespace